Generate Android Assets.jstalk

```diff
@@ -197,20 +231,24 @@ function process_slice(slice, doc, base_dir, factors) {
         var factor = factors[name];
         log("Processing " + slice_name + " " + name + " (" + factor + ")");
         var version = copy_layer_with_factor(slice, factor);
-        var file_name = base_dir + "/drawable-" + name + "/" + santize(slice_name) + ".png";
-        [doc saveArtboardOrSlice: version toFile:file_name];
-        log("Saved " + file_name);
+        var file_name_Drawable = base_dir + "/drawable-" + name + "/" + santize(slice_name) + ".png";
+        var file_name_Icon = base_dir + "/mipmap-" + name + "/" + santize(slice_name) + ".png";
+        if(isIconOrDrawable == "Icon"){
+            [doc saveArtboardOrSlice: version toFile:file_name_Icon];
+        }else{
+            [doc saveArtboardOrSlice: version toFile:file_name_Drawable];
+        }
+        
+        log("Saved ");
     }
 }
 
 function copy_layer_with_factor(original_slice, factor) {
     var copy = [original_slice duplicate];
     var rect = copy.className() == "MSArtboardGroup" ? rect = copy.absoluteRect().rect() : copy.absoluteInfluenceRect()
-    var request = MSExportRequest.new();
-    request.rect = rect;
-    request.scale = factor;
+    var slice = MSExportRequest.requestWithRect_scale(rect, factor)
```

Comment:
Whoops, looks like my previous commit was accidentally discarded here. `MSExportRequest.requestWithRect_scale()` has been removed from Sketch, so we need the lines from my previous commit here. Care to update the PR with this?